### DISCUSSION TITLE
Flux training has been proven to be feasible!

### DISCUSSION THREAD

**Discussion by leonary** [bghira/SimpleTuner#673] [General]
In the training of sd3, I found that it may not fit on a slightly larger data set. I am not sure what caused sd3 to fail to fit, but thanks to @bghira 's hard and amazing work, flux's Lora training has been verified to be feasible. Here are some results I got from fine-tuning with a very small data set:
![image](https://github.com/user-attachments/assets/2a726d43-8a5a-45ec-98a1-3e7d328ee847)
![image](https://github.com/user-attachments/assets/4dd2f6f1-b5df-465b-b6f1-1dd0fbdd15cb)
![image](https://github.com/user-attachments/assets/f394ce45-14af-48f7-8845-304cad3b8135)
![image](https://github.com/user-attachments/assets/e4b50e3a-0573-4c06-97d3-be62634af1e4)
![image](https://github.com/user-attachments/assets/cdaa2e46-df84-4f39-ba84-5a139f06ca85)
![image](https://github.com/user-attachments/assets/923a525c-542d-4f77-a624-4c4f9ea02eeb)

**Comment by bghira** [comment]
children, avert thy eyes! 🙈

**Comment by billnye2** [comment]
How many images were in your dataset and what hyperparameters did you use to train the lora?
